include/openssl/x509.h.in

```diff
@@ -281,6 +281,11 @@ typedef struct SCRYPT_PARAMS_st {
 } SCRYPT_PARAMS;
 #endif
 
+typedef struct x509ACert_st X509_ACERT;
+typedef struct x509ACertInfo_st X509_ACERT_INFO;
+typedef struct object_digest_info_st OBJECT_DIGEST_INFO;
+typedef struct IssuerSerial_st ISSUER_SERIAL;
```

Comment:
These identifiers and also the IETF_ATTR_SYNTAX* identifiers break the rule that new identifiers should be prefixed with ossl_ or OSSL_ prefix. Unfortunately it would make the identifiers even longer than they currently are. I suppose we want OTC to talk about this.